requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+pytest>=3.5.1
```

Comment:
Bind the versions, otherwise it would take latest always and something might fail. So better to use exact version or same minor version. That way it would always download a package on which the UTs are tested.

Reply:
Yeah, I know we do this for packages in requirements. But since this is dev dependencies, thought of picking up always new versions so that we get to use new features if any.

Reply:
Done.